tests/ops/qubit/test_special_unitary.py

```diff
@@ -741,6 +741,39 @@ def comparison(x):
         expected = tape.jacobian(loss, theta)
         assert np.allclose(jac, expected)
 
+    @pytest.mark.torch
+    @pytest.mark.jax
+    def test_large_wire_jacobian_regression(self, interface):
```

Comment:
Oops, my suggestion will have it run all of them like autograd 
```suggestion
    @pytest.mark.parametrize(interface, ["jax", "torch"])
    def test_large_wire_jacobian_regression(self, interface):
```